modules/swagger-codegen/src/main/resources/Java/libraries/feign/pom.mustache

```diff
@@ -161,7 +161,7 @@
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <swagger-core-version>1.5.8</swagger-core-version>
```

Comment:
@wing328 In your file, feign client still use 1.5.8. please have a check, https://github.com/swagger-api/swagger-codegen/blob/master/samples/client/petstore/java/feign/pom.xml#L163